python/cudnn/sdpa/fwd/engines.py

```
@@ -495,6 +499,92 @@ def _sm120_spec() -> EngineSpec:
+            padded_stats=True,
+            sched_policies=frozenset({SCHED_NATURAL}),
+            tile_ms=frozenset({64, 128}),
+            tile_ns=frozenset({64, 128}),
+            cgas=frozenset({1}),
+        ),
+        lower=partial(lower_dsl_prefill, api_type=_SM120),
+        knob_order=_sm120_fp8_knob_order,
+    )
+
+
+def knob_candidates(spec: EngineSpec, graph) -> List[Optional[SdpaFwdKnobs]]:
```

Ruff (0.16.1)

[error] Undefined name `List`

(F821)